doc/user/OPTIONS.md

```diff
@@ -142,3 +143,23 @@ Some rendering options are not compatible between them, here is the precedence o
 
 The `--options=value` syntax is used everywhere in this documentation, however, the syntax `--options value` can also be used, with the exception of options that have implicit values,
 `--verbose`, `--comp` and `--scalars`.
+
+## Filename templating
+
+The destination filename used by `--output` or to save screenshots can use the following template variables:
+
+- `{app}`: application name (ie. `F3D`)
+- `{version}`: application version (eg. `2.4.0`)
+- `{version_full}`: full application version (eg. `2.4.0-abcdefgh`)
+- `{model}`: current model filename without extension (eg. `foo` for `/home/user/foo.glb`)
+- `{model.ext}`: current model filename with extension (eg. `foo.glb` for `/home/user/foo.glb`)
+- `{model_ext}`: current model filename extension (eg. `glb` for `/home/user/foo.glb`)
+- `{date}`: current date in YYYYMMDD format
+- `{date:format}`: current date as per C++'s `std::put_time` format
+- `{n}`: auto-incremented number to make filename unique
```

Comment:
(up to 1000000)

Reply:
does it matter in the user documentation? The limit is only there because it's bad form to have unbound loops really. It's extremely unlikely to show up in nominal use.

Reply:
imo better safe than sorry. I think this is what @Meakk meant regarding documenting this though.